### DISCUSSION TITLE
Bash shell integration causes spurious '[' character before command output

### DISCUSSION THREAD

**Discussion by vhodges** [ghostty-org/ghostty#3760] [Feature Requests, Ideas] [answered]
As per the title, when bash shell integration is enabled (as is the default), an extra '[' character is output before the command is run.  I've disabled shell integration for now. I don't know if other shells have an issue or not.
I took a quick look at the files src/shell-integration/bash but I am no bash expert. I suspect one of the escape sequences is escaped when it shouldn't be or not escaped when it should be, but it wasn't obvious to me which one was causing the issue.  I even looked at the last commits for the two files, but it doesn't look like it was introduced recently.
If it matters I am running 1.0.1 via the Nix flake

**Comment by chipb** [comment, marked as answer]
This appears to be a dupe of #3566.

**Comment by vhodges** [comment]
Ah, sorry about that. I did search before posting but didn't come notice that post.
